python/cugraph/CMakeLists.txt

```diff
@@ -0,0 +1,88 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+cmake_minimum_required(VERSION 3.20.1 FATAL_ERROR)
+
+set(cugraph_version 22.08.00)
+
+file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-22.08/RAPIDS.cmake
+    ${CMAKE_BINARY_DIR}/RAPIDS.cmake)
+include(${CMAKE_BINARY_DIR}/RAPIDS.cmake)
```

Comment:
Since we're now fetching rapids-cmake in multiple places, I would switch to the model we have in cudf where we have a [single fetch-rapids.cmake file](https://github.com/rapidsai/cudf/blob/branch-22.08/fetch_rapids.cmake) and then [we include that file in other places that need it](https://github.com/rapidsai/cudf/blob/branch-22.08/cpp/CMakeLists.txt#L17).

Reply:
Addressed